docs/code/demo.py

```diff
@@ -6,19 +6,19 @@
 @app.task('daily')
 def do_daily():
     "This function runs once a day"
-    ...
+
 
 @app.task('daily between 07:00 and 10:00 | daily between 16:00 and 20:00')
 def do_twice_a_day():
     "This function runs twice a day (in the morning and in the afternoon)"
     # The '|' means OR operator. Fully supports logical operations.
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).